src/Languages/Python/Patterns/PyKeywordPattern.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Highlight\Languages\Python\Patterns;
+
+use Tempest\Highlight\IsPattern;
+use Tempest\Highlight\Pattern;
+use Tempest\Highlight\Tokens\TokenTypeEnum;
+
+final readonly class PyKeywordPattern implements Pattern
+{
+    use IsPattern;
+
+    public function __construct(private array $keywords = [
+        'and', 'as', 'assert', 'async', 'await', 'break', 'class', 'continue',
+        'def', 'del', 'elif', 'else', 'except', 'finally', 'for', 'from', 'global',
+        'if', 'import', 'in', 'is', 'lambda', 'nonlocal', 'not', 'or', 'pass',
+        'raise', 'return', 'try', 'while', 'with', 'yield',
+    ])
+    {
+    }
```

Comment:
Here I chose to add the keywords as an array directly within the class constructor for simplicity. For me this offers more efficiency and clarity, especially considering that the list of keywords is unlikely to change frequently.